jupyterhub/templates/_helpers.tpl

```diff
@@ -162,18 +162,6 @@ component: {{ include "jupyterhub.componentLabel" . }}
 {{- end }}
 
 
-{{- /*
-  jupyterhub.podCullerSelector:
-    Used to by the pod-culler to select singleuser-server pods. It simply
-    reformats "jupyterhub.matchLabels" and sets the componentLabel value so
-    `component=singleuser-server` is output.
-*/}}
-{{- define "jupyterhub.podCullerSelector" -}}
-{{- $_ := merge (dict "componentLabel" "singleuser-server") . -}}
-{{ include "jupyterhub.matchLabels" $_ | replace ": " "=" | replace "\n" "," | quote }}
-{{- end }}
-
-
```

Comment:
This helper was only used by the deleted pod-culler deployment

Reply:
The service running in the hub pod will now take care of the culling, and the hub pod has the credentials required do to so already.